draft-ietf-quic-http.md

```diff
@@ -1903,9 +1901,9 @@ The entries in the following table are registered by this document.
 | ----------------------------------- | ---------- | ---------------------------------------- | ---------------------- |
 | HTTP_NO_ERROR                       | 0x0000     | No error                                 | {{http-error-codes}}   |
 | HTTP_WRONG_SETTING_DIRECTION        | 0x0001     | Setting sent in wrong direction          | {{http-error-codes}}   |
-| HTTP_PUSH_REFUSED                   | 0x0002     | Client refused pushed content            | {{http-error-codes}}   |
+| Reserved                            | 0x0002     | N/A                                      | N/A                    |
```

Comment:
My sense is that we want to do a final cleanup for reserved error codes.  We might reserve some for keeping the relationship with RFC 7540 clean, but I don't think that a renumbering is a bad idea after all this work on fixing errors completes.

Reply:
Definitely. Not many good reasons exist to reserve codes for the shipped RFC.